### DISCUSSION TITLE
Feature Request: Linked Ingredients Show On Printed Recipe

### DISCUSSION THREAD

**Discussion by ChipSkylark37** [mealie-recipes/mealie#4765] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

If using the App / Website, and you have linked ingredients, “cook mode” shows you the linked ingredients. However, if printing the recipe, those linked ingredients do not show. 

### Please provide a concise description of the feature that would resolve your issue.

Sometimes I prefer to use a physical printout instead of keeping my phone open. This can be so I don’t get it messy, or I just don’t have my phone on me. I’d like to see which ingredients are linked to which step when having a physical printout. 

### Please consider and list out some caveats or tradeoffs made in your design decision

This could be an option when printing (adding to printing preferences). I see no downside or trade off with this being the case to have it toggled off by default to retain the current setting but being available for those that want it enabled. 

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [X] If this is accepted I'm willing to help maintain this feature
- [X] I'm willing to sponsor/pay a developer to do this work

**Comment by BWagener** [comment]
FYI I opened a PR which addresses this feature request https://github.com/mealie-recipes/mealie/pull/7632.
